### DISCUSSION TITLE
Reset not working with defaultChecked=true

### DISCUSSION THREAD

**Discussion by Vlad991** [react-hook-form/react-hook-form#8258] [Q&A]
Hi! Can you explain to me why "defaultChecked=true" breaks "reset" for this input. If I use { defaultValues } everything works. But for certain reasons, I need to set default values ​​​​using the input attribute. Here is [sandbox](https://codesandbox.io/s/practical-mopsa-7bkuz6).

**Comment by bluebill1049** [comment]
no easy way to overwrite the inline default checked, as it re-render the input that would take president on what to render.
